mobile-app/app/components/themed/ThemedIcon.tsx

```diff
@@ -1,11 +1,12 @@
 import { useThemeContext } from '@shared-contexts/ThemeProvider'
-import { MaterialCommunityIcons, MaterialIcons } from '@expo/vector-icons'
+import { MaterialCommunityIcons, MaterialIcons, Feather } from '@expo/vector-icons'
 import { IconProps } from '@expo/vector-icons/build/createIconSet'
 import { tailwind } from '@tailwind'
 
 import { ThemedProps } from './index'
 
-export type IconType = 'MaterialCommunityIcons' | 'MaterialIcons'
+export type IconType = 'MaterialCommunityIcons' | 'MaterialIcons' | 'Feather'
+export type IconName = React.ComponentProps<typeof MaterialIcons>['name'] | React.ComponentProps<typeof MaterialCommunityIcons>['name'] | React.ComponentProps<typeof Feather>['name']
```

Comment:
Wouldn't this break if there are common names across iconset?

Reply:
i believe repeated types will just get ignored 
<img width="368" alt="image" src="https://user-images.githubusercontent.com/40191153/175262364-a81c07f5-7cd9-4475-8152-2909fb1e88ec.png">

Reply:
It wont break any flow if there are common names coz we will be specifying `IconType` while using `IconName`. And its just type declaration, which I have placed at one place to avoid  repeated code